### DISCUSSION TITLE
这个置顶功能好像没什么用

### DISCUSSION THREAD

**Discussion by zhuhaisto** [tamlok/vnote#2178] [Q&A]
![置顶了但好像并没啥用](https://user-images.githubusercontent.com/108032980/177477862-09da8d5c-410c-432b-9c76-676f8a1e0491.png)

**Comment by tamlok** [comment]
这个排序是配置文件里面的排序，真正看到的顺序得看你的顶部那个查看顺序是什么。

**Reply by zhuhaisto** [reply]
那就是不能自定义排序了额？ 我肯定是想看笔记本的排序啊 不知道你这个排序有啥用

**Reply by tamlok** [reply]
你 查看 排序里面选择 按配置文件 就可以了。

**Reply by zhuhaisto** [reply]
> 你 查看 排序里面选择 按配置文件 就可以了。

懂了感谢

**Reply by zhuhaisto** [reply]
> 你 查看 排序里面选择 按配置文件 就可以了。

这个排序还是有问题  自定义设置（置顶了一个记录 其他的都是按最新创建排前面的） 然后查看的时候选择了自定义的排序规则是可以的 但是后面新增的条目它没有按最后新增的排到置顶的下方 而是直接排到了最下方

**Reply by tamlok** [reply]
> > 你 查看 排序里面选择 按配置文件 就可以了。
> 
> 这个排序还是有问题 自定义设置（置顶了一个记录 其他的都是按最新创建排前面的） 然后查看的时候选择了自定义的排序规则是可以的 但是后面新增的条目它没有按最后新增的排到置顶的下方 而是直接排到了最下方

新增默认就是放到配置文件里面最后面的。

**Reply by zhuhaisto** [reply]
> 

可我自定义设置 按最新的排最上，然后把一条置顶  这样按道理 那后面添加的所有 都是新增的排第二，置顶的还置顶 这样才符合逻辑

**Reply by tamlok** [reply]
> > 
> 
> 可我自定义设置 按最新的排最上，然后把一条置顶 这样按道理 那后面添加的所有 都是新增的排第二，置顶的还置顶 这样才符合逻辑

那个和查看排序不一样，那个是一次性的。

**Reply by zhuhaisto** [reply]
额 那至少能实现 按时间排序（最新添加的排最后）  然后再自定义一些条目的位置 然后查看的时候 新增的条目就会排到最后 这样看 貌似也有道理 符合逻辑 只是不能把最后加的排在最前 置顶的后面 反正我是这样理解了
